testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/OidcBaseTest.java

```diff
@@ -381,7 +429,20 @@ public static void loginToApp(String username, String password, int expectedStat
                 if (expectedText != null) {
                     String responseString = new BasicResponseHandler().handleResponse(afterLoginClickResponse);
                     assertTrue("Unexpected result " + responseString, responseString.contains(expectedText));
+                    if (expectedScope != null){
+                        assertTrue(context.toString().contains("scope=" + expectedScope));
+
+                        if(expectedScope.contains("profile")){
```

Comment:
This is another very minor nit, it's good to use a consistent formatting. To be consistent with the rest of the code, it's good to use spacing like this in if statements:


```
if (someCondition) {
    // do some stuff
} else {
   // do something else
}
```

Reply:
Thanks for pointing it out. The new if statements I added don't need to do anything for the else case. For consistency, should I do the following: 
```
if (someCondition) {
    // do some stuff
} else {
   // ignore
}
```

Reply:
No need to introduce else statements where they're not needed. Just wanted to show the overall format for future reference.

Reply:
Sounds good! I'll check to make sure I am following this format. Thanks

Reply:
fixed

Reply:
Thanks, I still seem to see `if(expectedScope.contains("profile")){` here. Maybe you have some local changes that haven't been pushed yet?